godtools/App/Services/Renderer/Views/Tool/Views/Card/ToolPageCardView.swift

```diff
@@ -177,8 +169,29 @@ class ToolPageCardView: MobileContentView {
         nextButton.isHidden = viewModel.hidesNextButton
     }
     
+    private func setupBottomGradientView() {
+        
+        bottomGradientView.isUserInteractionEnabled = false
+        bottomGradientView.backgroundColor = .clear
+        let bottomGradient = CAGradientLayer()
```

Comment:
Hey @reldredge71 ahh what's happening is when the bottomGradientView's bounds change (due to autolayout constraints) the added ```bottomGradient: CAGradientLayer``` doesn't update because constraints aren't applied to that CALayer.

What you have works, but it would be more optimized to just keep a reference to the added CAGradientLayer and then in layoutSubviews() method update the frame of the CAGradientLayer to match bounds of the bottomGradientView.

Also don't forget in the override layoutSubviews() to call super.